Makefile

```diff
@@ -24,28 +24,74 @@ endif
 
 MONOREPO_COMMIT=celo-core-contracts-v3.rc0
 
+# We checkout the monorepo as a sibling to the celo-blockchain dir because the
+# huge amount of files in the monorepo interferes with tooling such as gopls,
+# which becomes very slow.
+MONOREPO_PATH=../monorepo
```

Comment:
My only concern is this default. For my directory structure I already have `celo-monorepo` checkout out as `monorepo` as a sibling. I'd prefer something that was less likely to clash and made the purpose of the specific checkout more clear.

Reply:
Sure @celo-org/blockchain suggestions?

Reply:
I used `contracts_monorepo`, but that or `e2e_monorepo` or `e2e_contracts_monorepo`. I don't have too strong a preference on which variant.

Reply:
> For my directory structure I already have celo-monorepo checkout out as monorepo as a sibling.

Same here!  I'd be fine with whatever if it's unlikely to clash. It could even start with `.` (e.g. `.celo_blockchain_e2e_monorepo`). But also it might be nice to make it overridable with an environment variable.

Reply:
it's further down in the docs, but if you run make with `MONOREPO_PATH=$PATH` it will do the setup correctly (and then b/c it's symlinked, you don't need the env variable)

Reply:
Ok, I'm going for `.celo-blockchain_monorepo_checkout` unless anyone objects?

Reply:
I object to mixing `-` and `_`.  If we want celo-blockchain due to that being the repo name, might as well use dashes for all of them.